src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurDetailLineGroupForCollector.java

```diff
@@ -178,20 +260,56 @@ protected KualiDecimal calculateTotalAmountForLines(List<ConcurStandardAccountin
         return Optional.of(offsetEntry);
     }
 
+    protected Optional<OriginEntryFull> doNotBuildOffsetOriginEntry(

```

Comment:
This is being used as a method reference by earlier code. I'll update things accordingly to make the intent clearer.